src/CacheServiceErrorMapper.ts

```diff
@@ -24,7 +24,7 @@ export function cacheServiceErrorMapper(
     case Status.UNKNOWN:
     case Status.ABORTED:
     case Status.UNAVAILABLE:
-      return new InternalServerError(err?.message);
+      return new InternalServerError(`${err?.message}\n${err?.stack || ''}`);
```

Comment:
I think instead of string formatting, you can add a second param to `InternalServerError` and set the stack to `err?.stack`

https://github.com/momentohq/client-sdk-javascript/blob/295045a94ba24592b0257382f3986915670ab299/src/Errors.ts#L83

so it would become
```
export class InternalServerError extends CacheServiceError {
  constructor(message: string, stack?: string) {
    super(message);
    this.name = 'InternalServerError';
    this.stack = stack
  }
}
```